src/utils/report-uploader.js

```diff
@@ -9,8 +9,41 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
+import * as helixContentSDK from '@adobe/spacecat-helix-content-sdk';
+
 import { sleep } from '../support/utils.js';
 
+const SHAREPOINT_URL = 'https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/Shared%20Documents/sites/elmo-ui-data';
+
+/**
+ * @import { SharepointClient } from '@adobe/spacecat-helix-content-sdk/src/sharepoint/client.js'
+ */
+
+/* c8 ignore start */
+/**
+ * @param {object} context
+ * @param {object} context.env - Environment configuration object
+ * @param {string} context.env.SHAREPOINT_CLIENT_ID - SharePoint client ID
+ * @param {string} context.env.SHAREPOINT_CLIENT_SECRET - SharePoint client secret
+ * @param {string} context.env.SHAREPOINT_AUTHORITY - SharePoint authority URL
+ * @param {string} context.env.SHAREPOINT_DOMAIN_ID - SharePoint domain ID
+ * @param {Pick<Console, 'debug' | 'info' | 'warn' | 'error'>} context.log - Logger instance
+ * @returns {Promise<SharepointClient>}
+ */
+export function createLLMOSharepointClient({ env, log }) {
```

Comment:
[Suggestion] [Optional]should it be helix-utils instead of report utils.
Considering that we are working with helix api I think this might have more generic naming for future reuse.
Also, long run I think we should use base_url from the config to have separation between prod/stage(I don't know if it exists)

Reply:
yeah I know. Initially, I put it into an extra file, but all the related functionality was already here. For that reason I kept it here.